ci_framework/roles/env_op_images/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+  when:
+    - openstack_installed.stdout == "True"
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ignore_errors: true
+  block:
+    - name: Create a file to store the images
+      ansible.builtin.copy:
+        dest: "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+        content: "cifmw_install_yamls_vars:"
+
+    - name: Get openstack operator-index image
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          pod_name=$(oc get pods -l olm.catalogSource=openstack-operator-index --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}');
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+          echo -e "\n  OPENSTACK_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get openstack-operator image
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          pod_name=$(oc get pods -l openstack.org/operator-name=openstack --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}' );
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].imageID}');
+          echo -e "\ncifmw_openstack_operator_image:\n  OPENSTACK_OP_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get Rabbitmq-operator image
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          pod_name=$(oc get pods -l app.kubernetes.io/component=rabbitmq-operator --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | awk '{print $1}');
+          image=$(oc get pod ${pod_name} -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[0].imageID}');
+          echo "  RABBITMQ_OP_IMG: ${image}" >> "{{ cifmw_env_op_images_dir }}/artifacts/{{ cifmw_env_op_images_file  }}"
+
+    - name: Get rest of the operator images
+      ci_script:
+        output_dir: "{{ cifmw_env_op_images_dir }}/artifacts"
+        script: |
+          for pod in $(oc get pod -l openstack.org/operator-name --no-headers=true -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} | grep -v 'openstack-operator-controller-manager' | awk '{print $1}');  do
+             operator_name=$(oc get pod $pod -o jsonpath='{.metadata.labels.openstack\.org/operator-name}' -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} --no-headers=true | tr '[:lower:]' '[:upper:]');
+             image_name=$(oc get pod $pod -n {{ cifmw_install_yamls_defaults['OPERATOR_NAMESPACE'] }} -o jsonpath='{.status.containerStatuses[1].image}' );
```

Comment:
Why we fetch here the full ID (sha265 one) and in the rest of the examples before we fetch the plain name and tag?